apps/web/app/api/v1/client/actions/route.ts

```diff
@@ -0,0 +1,49 @@
+import { transformErrorToDetails } from "@/app/lib/api/validator";
+import { responses } from "@/app/lib/api/response";
+import { createAction } from "@formbricks/lib/action/service";
+import { ZActionInput } from "@formbricks/types/actions";
+import { NextResponse } from "next/server";
+
+export async function OPTIONS(): Promise<NextResponse> {
+  return responses.successResponse({}, true);
+}
+
+export async function POST(req: Request): Promise<NextResponse> {
+  try {
+    const jsonInput = await req.json();
+
+    // validate using zod
+    const inputValidation = ZActionInput.safeParse(jsonInput);
+
+    if (!inputValidation.success) {
+      return responses.badRequestResponse(
+        "Fields are missing or incorrectly formatted",
+        transformErrorToDetails(inputValidation.error),
+        true
+      );
+    }
+
+    const { environmentId, name, properties, personId } = inputValidation.data;
+
+    // hotfix: don't create action for "Exit Intent (Desktop)", 50% Scroll events
+    if (["Exit Intent (Desktop)", "50% Scroll"].includes(name)) {
```

Comment:
if this are the new endpoints we can remove the hotfix as long as we make sure in the current formbricks-js code that these are not sent